src/collection.js

```diff
@@ -537,6 +590,7 @@ export default class Collection {
         if (!syncResultObject.ok) {
           return syncResultObject;
         }
+        // XXX import of data should be done after conflict handling.
```

Comment:
That makes sense, but is probably broader than the scope of this patch; could you please file an issue about it?